### ISSUE TITLE
More Pokemon info to be displayed

### ISSUE THREAD

**Issue by popsoda27** [GediminasMasaitis/PokeEase#54]
Is it possible to extract more info from captured pokemons to display? Certain fields like HP, amount of stardust to level up, amount of candies to level up, amount of candies to evolve

**Comment by GediminasMasaitis** [comment]
Yes, we do have more data to show. We just need to come up with a design for extended info.
